app/js/routes.jsx

```diff
@@ -0,0 +1,23 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import { AuthenticatedRoute, LoginPage, Header } from '@openmrs/react-components';
+
+import App from './components/App';
+
+export default store => (
+  <div>
+    <Header />
+    <Switch>
+      <Route path="/login" component={LoginPage} />
+      <AuthenticatedRoute path="/" component={App} />
```

Comment:
This OWA is going to live within an existing OpenMRS application, like Order Entry, so  we don't need the Login Page or to use the "Authenicated Route"...  the main OpenMRS application it runs within should handle making sure the user is authenticated, tracking the user's session, and redirecting to the main OpenMRS login page if needed.

We can get rid of the "/login" route and the top-level route can be a plain Route instead of an AuthenticatedRoute.

(Although, interestingly, we may want to use this app outside of the main OpenMRS app at some point, but we can look into adding this in at that time).